Src/Base/AMReX_PODVector.cpp

```diff
@@ -0,0 +1,13 @@
+#include <AMReX_PODVector.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_REAL.H>
+
+namespace amrex::VectorGrowthStrategy
+{
+    Real growth_factor = 1.5_rt;
+
+    void Initialize () {
+        ParmParse pp("amrex");
+        pp.queryAdd("vector_growth_factor", growth_factor);
```

Comment:
We probably should check it's not less (1+epsilon) and not greater a huge number.

Reply:
Good idea.